.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'Type: Bug'
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Code reproducing the issue**
+If applicable, add trimed down, self-contained code demonstrating the issue.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+** Environment and Version info (please complete the following information):**
+ - OS: [e.g. Windows 10, macOS 12 Monterey, Ubuntu 22.04]
+ - Version [e.g. 2021.01, 2023.04.16]
```

Comment:
Perhaps add that for graphics-related issued, include GL info from the info menu?

Reply:
Yes, good idea.